packages/documentation/.storybook/toolbar/story-container-base.ts

```diff
@@ -0,0 +1,26 @@
+import { Args, StoryContext, StoryFn } from '@storybook/web-components';
+import { LitElement, unsafeCSS } from 'lit';
+import { property } from 'lit/decorators.js';
+
+/**
+ * Basis for story container web components that render a story inside a shadow DOM.
+ */
+export class StoryContainerBase extends LitElement {
+  @property({ attribute: true })
+  story!: StoryFn;
+
+  @property({ attribute: true })
+  context!: { args: StoryContext<Args> };
+
+  render() {
+    return this.story(this.context, this.context.args);
+  }
+}
+
+/**
+ * Transforms a style input into a CSSResult output adapted to web components
+ */
+export function useStyles(styles: any) {
+  if (typeof styles !== 'string') return unsafeCSS('');
+  return unsafeCSS(styles.replace(/:root/g, ':host'));
```

Comment:
Did you try to change the `:root` selector to `:scope` in the tokens core.scss file?
I thought, this was a topic we discussed last week?

Ignore this if it just did not work ;)

Reply:
I added a comment about that but it was pending... :sweat_smile:
Here it is: https://github.com/swisspost/design-system/pull/3441#discussion_r1727173846